cmake/Utils.cmake

```diff
@@ -19,6 +19,10 @@ function(add_onnx_global_defines target)
   if(ONNX_USE_LITE_PROTO)
     target_compile_definitions(${target} PUBLIC "ONNX_USE_LITE_PROTO=1")
   endif()
+
+  if(ONNX_DISABLE_STATIC_REGISTRATION)
+    target_compile_definitions(${target} PUBLIC "__ONNX_DISABLE_STATIC_REGISTRATION=1")
```

Comment:
do you need =1?

Reply:
I followed the same logic as ONNX_ML and ONNX_USE_LITE_PROTO, but yes I think pure `__ONNX_DISABLE_STATIC_REGISTRATION` should be also workable. Let's try it.